src/runner/main.cpp

```diff
@@ -273,7 +349,7 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
             return 0;
         }
     case SpecialMode::ReportSuccessfulUpdate:
-        notifications::show_toast(GET_RESOURCE_STRING(IDS_AUTOUPDATE_SUCCESS));
+        OnSuccessfulUpdate();

```

Comment:
Let's rename it to `RequestExplorerRestart`

Reply:
Renamed.